components/FilterButton/FilterButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import styles from './FilterButton.module.scss';
+
+const FilterButton = () => {
+    const [filterStatus, setFilterStatus] = useState("all");
+
+    const handleFilterButtonClick = () => {
+        switch (filterStatus) {
+            case "all":
+                setFilterStatus("deposit");
```

Comment:
@Pieter-Ixo I initially set this FilterButton to work in the same component as where the data is being queried and rendered, I still need to edit it so that it can work separately, for cleaner code.